### PR TITLE
Quickstart sample

### PR DESCRIPTION
**Pull request by OzBenDev**

### CODE REVIEW

File: SDKSamples/common/build.gradle [hunk 1]
```diff
@@ -70,31 +70,31 @@ dependencies {
...
 
-    api 'androidx.appcompat:appcompat:1.3.0-rc01'
-    api 'androidx.recyclerview:recyclerview:1.1.0'
+    api 'androidx.appcompat:appcompat:1.4.0-beta01'
+    api 'androidx.recyclerview:recyclerview:1.2.1'
 
     api 'com.google.android.material:material:1.2.0' // upgrade cause messed up chat bubbles
 
-    api 'androidx.constraintlayout:constraintlayout:2.0.4'
+    api 'androidx.constraintlayout:constraintlayout:2.1.1'
```
Comment (tehilaR2) [inline SDKSamples/common/build.gradle:89]: is it safe?
previously when done the chat elements appeared cut

File: SDKSamples/build.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.4.30'
+    ext.kotlin_version = '1.5.31'
```
Comment (tehilaR2) [inline SDKSamples/build.gradle]: is it safe?
when i tried to upgrade i had compile issues

**Review by tehilaR2 — COMMENTED** [review]
please reply to the comments

File: SDKSamples/common/build.gradle (hunk 1 above)
Comment (OzBenDev) [reply SDKSamples/common/build.gradle:89]: it works for me.
ill downgrade  it to be safe.

File: SDKSamples/build.gradle (hunk 2 above)
Comment (OzBenDev) [reply SDKSamples/build.gradle]: updated

**Review by tehilaR2 — APPROVED** [review]

**Pull request merged**